modules/cloud-config-container/simple-nva/variables.tf

```diff
@@ -14,6 +14,18 @@
  * limitations under the License.
  */
 
+variable "bgp_config" {
```

Comment:
please update copyright at the beginning of file

Reply:
Updated!